src/helpers/Auth.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import Cookies from 'js-cookie';
+import { Requester } from './Requests';
+
+export const COOKIE_LOGIN_KEY = 'ua_session_token';
```

Comment:
I would prefer more specific/obvious key, e.g. `faq_ua_session_token`

Reply:
Are you referring to the value of the cookie key? ie.`ua_session_token`.

If so, I think we should keep it as that since this is the one kiwicom uses and otherwise we wouldn't share the same session with them.

Reply:
oh, I see, ok then